### PR TITLE
Fixed issue in CopyFiles task for directories with square brackets #9589

### PR DESCRIPTION
**Pull request by jtpetty**
The issue was that the path was ultimately sent to minimatch for matching against a list of files to determine inclusion. Square brackets, [], are special characters for minimatch patterns and needed to be escaped prior to calling tl.match()

### CODE REVIEW

File: Tasks/CopyFilesV2/Strings/resources.resjson/en-US/resources.resjson [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "loc.friendlyName": "Copy files",
-  "loc.helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkID=708389)",
+  "loc.helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?LinkID=708389)",
```
Comment (stephenmichaelf) [inline Tasks/CopyFilesV2/Strings/resources.resjson/en-US/resources.resjson]: Was this change intentional?

Comment (jtpetty) [reply Tasks/CopyFilesV2/Strings/resources.resjson/en-US/resources.resjson]: It happened automatically when I built. Seemed harmless so I left it.
